### PR TITLE
feat: add php8.2 image

### PR DESCRIPTION
**Pull request by DeepDiver1975**
required for https://github.com/owncloud/core/pull/40981

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
images php7.0-php7.3 are not building

**Comment by phil-davis** [comment]
Looks like some old-version PHP has "gone away" in conjunction with whatever Ubuntu we are trying to install it on...

We should be able to just remove all that. We do not need PHP 7.3 or earlier in any CI any more.

And I think that if we remove the old PHP build pipelines, the existing docker images for those versions will remain being published as-they-are. Is that correct @xoxys ?

**Comment by DeepDiver1975** [comment]
moved this to a dedicated issue  unrelated to this pr.

**Comment by DeepDiver1975** [comment]
approval and merge is welcome to unblock my core pr - thx

File: v8.2/manifest.tmpl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+image: owncloudci/php:8.1
```
Comment (xoxys) [inline v8.2/manifest.tmpl:1]: Still `8.1`

**Review by xoxys — CHANGES_REQUESTED** [review]
Some files need to be renamed to 8.2:

image

**Review by xoxys — APPROVED** [review]

**Comment by DeepDiver1975** [comment]
THX

**Pull request merged**